clients/edgex-foundry/util.go

```diff
@@ -50,29 +54,28 @@ func getEdgeDeviceName(d *devicev1alpha1.Device) string {
 	return actualDeviceName
 }
 
-func toEdgexDeviceService(ds *devicev1alpha1.DeviceService) models.DeviceService {
-	return models.DeviceService{
-		Description: ds.Spec.Description,
-		Name:        ds.GetName(),
-		//Id:             ds.Spec.Id,
+func toEdgexDeviceService(ds *devicev1alpha1.DeviceService) dtos.DeviceService {
+	return dtos.DeviceService{
+		Description:   ds.Spec.Description,
+		Name:          ds.GetName(),
 		LastConnected: ds.Status.LastConnected,
 		LastReported:  ds.Status.LastReported,
 		Labels:        ds.Spec.Labels,
-		AdminState:    models.AdminState(ds.Spec.AdminState),
+		AdminState:    string(models.AdminState(ds.Spec.AdminState)),
```

Comment:
There seems to be a redundant type conversion here, how about changing it directly to `string(ds.Spec.AdminState)`?